server/models/StageModel.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose')
+
+const stageSchema = new mongoose.Schema({
+  _id: {
+    type: String,
+    required: true,
+  },
+  dayId: {
+    type: String,
+    required: true,
+  },
+  tripId: {
+    type: String,
+    required: true,
+  },
+  dayIndex: {
+    type: Number,
+    required: true,
+  },
+  stageIndex: {
+    type: Number,
+    required: true,
+  },
+  stagesLongitude: Number,
```

Comment:
maybe just stageLongitude since this is for just one stage?

Reply:
Fixed!